torch/csrc/jit/fuser/cuda/thnvrtc.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <cuda.h>
+#include <nvrtc.h>
+
+// See [USE OF NVRTC AND DRIVER API]
+
+#define TORCH_FORALL_NVRTC(_)                    \
+  _(nvrtcVersion)                                \
+  _(nvrtcCreateProgram)                          \
+  _(nvrtcDestroyProgram)                         \
+  _(nvrtcGetPTXSize)                             \
+  _(nvrtcGetPTX)                                 \
+  _(cuModuleLoadData)                            \
+  _(cuModuleGetFunction)                         \
+  _(cuOccupancyMaxActiveBlocksPerMultiprocessor) \
+  _(cuGetErrorString)                            \
+  _(nvrtcGetErrorString)                         \
+  _(cuLaunchKernel)                              \
+  _(nvrtcCompileProgram)                         \
+  _(cuCtxGetCurrent)                             \
+  _(cuModuleUnload)
+
+extern "C" typedef struct THNVRTC {
+#define CREATE_MEMBER(name) decltype(&name) name;
```

Comment:
is this really necessary? seems like copypasta `decltype(&foo) foo;` is not so bad?

Reply:
there is also another assignment thing in the .cpp file and it would be easy to miss that when adding a new function.